scheduler/project.clj

```diff
@@ -13,7 +13,7 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 ;;
-(defproject cook "1.5.3-SNAPSHOT"
+(defproject cook "1.5.3"
```

Comment:
I would favor having this move to `1.5.4-SNAPSHOT`, with a commit containing `1.5.3` that we can tag for the release (and then not squash the commits)